.github/workflows/continuous-integration.yml

```diff
@@ -27,7 +22,7 @@ jobs:
       - uses: actions/checkout@v4
       - uses: p0deje/setup-bazel@0.9.0
         with:
-          bazelrc: common --announce_rc --color=yes --enable_bzlmod=${{ matrix.bzlmodEnabled }}
```

Comment:
I imagine we can do some cleanup in the root module if it's no longer built with WORKSPACE, I'll take that as a follow-up